submissions/Yasya23/holi-style-popup/style.css

```diff
@@ -0,0 +1,179 @@
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+:focus {
```

Comment:
pseudo-element without element?

Reply:
This element removes the default focus everywhere so that it is not like in the example. 

![Focus](https://github.com/Yasya23/Hooli-style-Popup/blob/main/img/focus.jpg)